src/library/playlisttablemodel.cpp

```diff
@@ -272,24 +270,24 @@ bool PlaylistTableModel::isColumnHiddenByDefault(int column) {
 }
 
 TrackModel::CapabilitiesFlags PlaylistTableModel::getCapabilities() const {
-    TrackModel::CapabilitiesFlags caps = TRACKMODELCAPS_NONE
-            | TRACKMODELCAPS_RECEIVEDROPS
-            | TRACKMODELCAPS_REORDER
-            | TRACKMODELCAPS_ADDTOCRATE
-            | TRACKMODELCAPS_ADDTOPLAYLIST
-            | TRACKMODELCAPS_EDITMETADATA
-            | TRACKMODELCAPS_LOADTODECK
-            | TRACKMODELCAPS_LOADTOSAMPLER
-            | TRACKMODELCAPS_LOADTOPREVIEWDECK
-            | TRACKMODELCAPS_RESETPLAYED;
+    TrackModel::CapabilitiesFlags caps =
+            TRACKMODELCAPS_RECEIVEDROPS |
+            TRACKMODELCAPS_REORDER |
+            TRACKMODELCAPS_ADDTOCRATE |
+            TRACKMODELCAPS_ADDTOPLAYLIST |
+            TRACKMODELCAPS_EDITMETADATA |
+            TRACKMODELCAPS_LOADTODECK |
+            TRACKMODELCAPS_LOADTOSAMPLER |
+            TRACKMODELCAPS_LOADTOPREVIEWDECK |
+            TRACKMODELCAPS_RESETPLAYED;
 
     if (m_iPlaylistId != m_pTrackCollectionManager->internalCollection()->getPlaylistDAO().getPlaylistIdFromName(AUTODJ_TABLE)) {
         // Only allow Add to AutoDJ if we aren't currently showing the AutoDJ queue.
         caps |= TRACKMODELCAPS_ADDTOAUTODJ | TRACKMODELCAPS_REMOVE_PLAYLIST;
     } else {
         caps |= TRACKMODELCAPS_REMOVE;
     }
-    if (m_pTrackCollectionManager->internalCollection()->getPlaylistDAO().getHiddenType(m_iPlaylistId)== PlaylistDAO::PLHT_SET_LOG) {
+    if (m_pTrackCollectionManager->internalCollection()->getPlaylistDAO().getHiddenType(m_iPlaylistId) == PlaylistDAO::PLHT_SET_LOG) {
```

Comment:
long line

Reply:
A line break here looks a little weird:
![image](https://user-images.githubusercontent.com/13354331/82544885-1892fb80-9b56-11ea-87b4-2f17953fccd8.png)

Reply:
But I fixed it in https://github.com/mixxxdj/mixxx/pull/2715/commits/bd4a67d8007602eccfc0b224d4ad94b99a6abb0b#diff-f7dd1ed098b395653d5cdc492c7f7b20 :)